### ISSUE TITLE
Make it possible to play own playlists

### ISSUE THREAD

**Issue by nicolaiskogheim** [hnarayanan/shpotify#26]
`spotify play list mylist` is useless if I can't play my own lists.
I read somewhere that the chosen list gets picked at random. That's fine, but I would like shpotify to search my own lists first, and give priority to those.

(I must say that the random list picking have given me some wonderful song results, so I am not at all angry about this, but, you know, it would be nice to be able to play ones lists)

**Comment by hnarayanan** [comment]
Please submit a patch. 👍

**Comment by hnarayanan** [comment]
There exists a pull request that might resolve this. Could you test it and report if it satisfies your needs?

https://github.com/hnarayanan/shpotify/pull/27

**Comment by nicolaiskogheim** [comment]
It does not :) I want to play my own playlists. If I have to open the spotify app to copy the URI and then paste it in the terminal.. you get the point.

**Comment by nivv** [comment]
There is an endpoint that fetches a list of the current user's playlist. https://developer.spotify.com/documentation/web-api/reference/playlists/get-a-list-of-current-users-playlists/

**Comment by davidem** [comment]
@nivv : you're correct, there's an endpoint to list the current user's playlist through https://api.spotify.com/v1/me/playlists, but, it requires authorisation. 

I've added exactly that to my fork with pull request #131. I can now list my playlists and use the uri to play any of that playlists :)
